catalogs/example_catalog_4.json

```diff
@@ -0,0 +1,1050 @@
+[
+  {
+    "id": "Example Catalog 4",
+    "name": "Example Catalog 4",
+    "description": "This is an example catalog for testing purposes. The catalog contains 3 categories with controls, sub-controls, metrics and all_in_scope disabled.",
+    "categories": [
```

Comment:
No field for `"all_in_scope": false`?

Reply:
If it is not set, it is automatically false. If you like, we can also add that.